Swith to a pure python version of Fernet

geffrak opened the pull request:
DESCRIPTION

In order to make the packaging easier and to have a predictable result,
we switch from the cryptography implementation of Fernet to a custom
python based one.

jfboismenu commented:
Also, this fork has Python 2.7 support in it. Maybe it's worth forking that repo and using it?
https://github.com/sahilbathlaofficial/python-fernet

geffrak commented:
This is mostly the changes I made. Will fork it in shotgunsoftware

geffrak commented:
@jfboismenu I changed the requirements.txt file [here](https://github.com/shotgunsoftware/tk-framework-desktopclient/blob/dev/geffrak/python-only-vendors/requirements.txt) ([previously](https://github.com/shotgunsoftware/tk-framework-desktopclient/blob/master/requirements.txt))

So now since there's no need for the comlicated vendor folder structure of `Vendors/{os}/{python_version} I updated the vendor scrips to make a flat Vendor folder [here](
https://github.com/shotgunsoftware/tk-framework-desktopclient/blob/dev/geffrak/python-only-vendors/vendors.sh) ([previously](https://github.com/shotgunsoftware/tk-framework-desktopclient/blob/master/vendors.sh)) and the code that parse it [here](https://github.com/shotgunsoftware/tk-framework-desktopclient/blob/dev/geffrak/python-only-vendors/framework.py#L20-L33) ([previously](https://github.com/shotgunsoftware/tk-framework-desktopclient/blob/master/framework.py#L20-L42)).
